lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1145,12 +1155,42 @@ def view_init(self, elev=None, azim=None, roll=None, vertical_axis="z",
 
         self._dist = 10  # The camera distance from origin. Behaves like zoom
 
-        if elev is None:
-            elev = self.initial_elev
+        def update_arg(arg, old_arg):
```

Comment:
I like the structure of this change! Could you add a test for the positional args case?

Reply:
A test has been added (test_axes3d.py::test_view_init_positional_args).